web-client/src/presenter/utilities/downloadXlsx.test.ts

```diff
@@ -0,0 +1,89 @@
+import { downloadXlsx } from '@web-client/presenter/utilities/downloadXlsx';
+
+const oldWindowUrl = window.URL;
+const oldCreateElement = window.document.createElement;
+
+const encodedXlsxArray = [65, 66, 67];
+const fileName = 'fileName';
+const mockUrl = 'www.test.com';
+
+let anchorElement;
+let createElementSpy;
+let createObjectURLSpy;
+let revokeObjectURLSpy;
+
+describe('downloadXlsx', () => {
+  beforeEach(() => {
+    anchorElement = window.document.createElement('a');
+    jest.spyOn(anchorElement, 'click');
+    jest.spyOn(anchorElement, 'remove');
+    jest.spyOn(anchorElement, 'setAttribute');
+
+    createElementSpy = jest.fn().mockReturnValue(anchorElement);
+    window.document.createElement = createElementSpy;
+
+    createObjectURLSpy = jest.fn().mockReturnValue(mockUrl);
+    revokeObjectURLSpy = jest.fn();
+
+    window.URL.createObjectURL = createObjectURLSpy;
+    window.URL.revokeObjectURL = revokeObjectURLSpy;
+  });
+
+  afterEach(() => {
+    window.URL = oldWindowUrl;
+    window.document.createElement = oldCreateElement;
+    jest.restoreAllMocks();
+  });
+
+  it('creates a blog and generates an object URL', () => {
```

Comment:
Small (but entertaining!) typo: `blog` -> `blob`

Also not sure if we want the "_should_ create ... " language in these tests. I'm not a stickler for these things.